package.json

```diff
@@ -41,7 +41,7 @@
   "peerDependencies": {
     "cldr-data": ">=25",
     "globalize": "^1.1.0-rc.5 <1.3.0",
-    "webpack": "^2.5.1"
+    "webpack": "^2.2.0-rc"
```

Comment:
> Additionally, the required webpack version here is downgraded to match what is required by react-globalize-webpack-plugin.

I don't get it... If the consumer product is using webpack 2.5.1 it would satisfy both GWP and RGWP, wouldn't it?